todo_app/templates/index.html

```diff
@@ -10,9 +10,22 @@ <h1 class="display-4">To-Do App</h1>
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
-      <ul class="list-group mb-4">
+      <ul class="list-group mb-4">        
         <!-- List the todo items here -->
+        {% for item in items %}
+        <li>{{item.title}}</li>
+        {% endfor %}
       </ul>
     </div>
+    <form action="" method="post" class="submitform">
```

Comment:
Your app seems to expect this form to post to `/new`, so we should specify that as the action here